app/services/locale_chooser.rb

```diff
@@ -16,6 +16,6 @@ def locale
   attr_reader :locale_param, :request
 
   def locale_valid?
-    LocaleValidator.new(locale_param).success?
```

Comment:
This class was small and only used here, so @zachmargolis and I made the executive decision to reduce misdirection by removing it and copying the logic here.